lib/genome/groupers/interaction_grouper.rb

```diff
@@ -21,49 +24,59 @@ def self.reset_members
       end
 
       def self.add_members
-        DataModel::InteractionClaim.joins(drug_claim: [:drug], gene_claim: [:gene]).where(interaction_id: nil).each do |interaction_claim|
-          drug = interaction_claim.drug_claim.drug
-          gene = interaction_claim.gene_claim.gene
-          interaction = DataModel::Interaction.where(drug_id: drug.id, gene_id: gene.id).first_or_create
-          interaction_claim.interaction = interaction
-          interaction_claim.interaction_claim_types.each do |t|
-            unless interaction.interaction_types.include? t
-              interaction.interaction_types << t
+        DataModel::InteractionClaim.eager_load(:interaction_claim_types, :source, :interaction_claim_attributes, :publications).joins(drug_claim: [:drug], gene_claim: [:gene]).where(interaction_id: nil).each do |interaction_claim|
```

Comment:
This isn't a change from your PR per se but I'm worried now that we're eager loading everything if we run this from scratch, if loading _all_ the interaction claims at once might be too much memory? We could change this `each` into `in_batches` (https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-in_batches) without any further code changes to avoid this.

Reply:
I made this change.